tiled/commandline/_serve.py

```diff
@@ -98,6 +98,9 @@ def serve_directory(
         ),
     ),
     port: int = typer.Option(8000, help="Bind to a socket with this port."),
+    log_config: Optional[str] = typer.Option(
+        None, help="Custom uvicorn logging configuration file"
```

Comment:
Uvicorn provides logging config out of the box for tiled, but is this the first time we expose this internal detail through a "public" interface?

I suppose there are no plans to move to / support a different ASGI server for the CLI...so just pointing out that this is stepping out from under the hood.

Reply:
Good eye. We do have a univorn section in the server config schema too, so this is not the first.

Worth reviewing later whether we have a good enough story for other ASGI servers.